container_files/public_html/doc/builtin/procedures/DistTableProcedure.md

```diff
@@ -0,0 +1,75 @@
+# Stats Table Procedure
+
+For a combination of feature columns (the `training_data` parameter) and outcomes, this procedure
+will iteratively (over the rows) calculate different statistics (count, avg, std, min, max) on
+the outcomes, for each values of the feature columns. 
+
+The feature columns are assumed to be string-valued, and the outcome numeric-valued (type `INTEGER` or `NUMBER`). 
+
+This procedure can be views as a ![](%%doclink statsTable.bagOfWords.train procedure) where the
+labels are numbers instead of booleans.
+
+You can optionally specity an output dataset that will contain the computed rolling statistics for each feature column / outcome
+combination. Since we are doing it iteratively, it means that the statistics for row `n` will only use the `n-1` preceding rows.
```

Comment:
Nah pcq "for each" mais c'est pas clair ce que mon slash veut dire, je vais ajouter des guillemets:
for each "feature column / outcome" combination?